### ISSUE TITLE
Set stime, retime, and rutime values

### ISSUE THREAD

**Issue by elazar125** [azielonko/xv6-public#17]

**Comment by azielonko** [comment]
not sure if you care at this point, but the cpu-bound sanity tests seem to be borked. i can set the number of loops to something ridiculous, and it still completes almost instantaneously. for reference, i tried running the same tests in just a regular C program, and it's taken about 3 mins so far. this would also explain why the cpu-bound stats give us really low values every time

**Comment by elazar125** [comment]
I honestly don't care at this point, as I need to wake up for work in three hours. However, did you notice similar results when yielding as to plain old noops?

**Comment by azielonko** [comment]
yeah, those were pretty similar

**Comment by elazar125** [comment]
If you check out my results, it's not necessarily 0 across the board - there are some non-zero sleep and ready times. I wonder if maybe we're just checking what the process did in the wrong way.

**Comment by azielonko** [comment]
maybe, maybe not. it really depends on what's going on with the cpu tests. it's kind of hard to tell at this point. i'll keep poking around and update the readme if i end up with something that looks like normal-ish results

**Comment by elazar125** [comment]
Sounds good. If you don't mind, I'd like to go to sleep. I'll bring my laptop to work, so I can periodically check on what's going on with this project before we submit.

**Comment by azielonko** [comment]
yeah for sure. gnight. and thanks for being a good player through all of this. i know i definitely didn't carry my fair share of work for this assignment. i owe you one
